1_space_odyssey/index.html

```diff
@@ -10,7 +10,7 @@
 </head>
 <body class="fade-in">
     
-    <script src="./js/mannuel-9000/app.js"></script>
+    <script src="./js/mannuel-9000/henrik-9000/app.js"></script>
```

Comment:
please move your folder into the JS folder and not my named folder ;-)
`js/henrik-9000/app.js`

Reply:
@henrikaag please update your script tag to look like this:
`<script src="./js/henrik-9000/app.js"></script>`